spec/models/list_import/my_anime_list/row_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe ListImport::MyAnimeList::Row do
+  context 'with anime' do
+    let(:anime) { fixture('list_import/my_anime_list/toy-anime.json') }
```

Comment:
Yeah overlooked that. I guess this will work not happy about the let statement where it is but w/e.